visual-inspection/inspection-backend/inspection/model/model.py

```diff
@@ -1,21 +1,70 @@
 import tensorflow as tf
 
 # TODO: Replace with custom model variant
-model = tf.keras.applications.MobileNetV2(input_shape=None,
-                                          include_top=True,
-                                          weights="imagenet")
-
+#model = tf.keras.applications.MobileNetV2(input_shape=None,
+#                                          include_top=True,
+#                                          weights="imagenet")
+model = tf.keras.applications.Xception( include_top=True,
+                                        weights="imagenet",
+                                        classes=1000,
+                                        classifier_activation="softmax")
 GERMAN_LABELS = {
+    "Polaroid_camera": "eine Sofortbildkamera",
+    "lens_cap": "ein Objektivdeckel",
+    "pedestal": "ein Sockel",
+    "necklace": "eine Halskette",
+    "typewriter_keyboard": "eine Schreibmaschinentastatur",
+    "laptop": "ein Laptop",
+    "pitcher": "ein Krug",
+    "tray": "ein Tablett",
+    "dishrag": "ein Abwaschtuch",
+    "doormat": "eine Fußmatte",
+    "binder": "ein Bindemittel",
+    "pencil_box": "eine Federtasche",
+    "joystick": "ein Steuergriff",
+    "letter_opener": "ein Brieföffner",
+    "measuring_cup": "ein Messbecher",
+    "mortar": "ein Mörtel",
+    "rubber_eraser": "ein  Radierer",
+    "projector": "ein Projektor",
+    "buckle": "eine Schnalle",
     "reel": "eine Rolle",
+    "paintbrush": "ein Malpinsel",
+    "ocarina": "eine Okarina",
+    "bottlecap": "Flaschenverschluss",
+    "syringe": "eine Spritze",
+    "lotion": "eine Lotion",
+    "stole": "eine Stola",
+    "apron": "eine Schürze",
+    "mailbag": "ein Briefbeutel",
+    "perfume": "ein Parfüm",
+    "sandal": "eine Sandale",
+    "flute": "eine Flöte",
+    "binoculars": "ein Fernglas",
+    "book_jacket": "ein Buchumschlag",
+    "honeycomb": "eine Honigwabe",
+    "poncho": "ein Poncho",
+    "water_bottle": "eine Wasserflasche",
+    "padlock": "ein Vorhängeschloss",
+    "pajama": "ein Schlafanzug",
+    "scale": "ein Maßstab",
+    "bib": "ein Latz",
+    "cassette_player": "ein Kassettenspieler",
+    "kimono": "ein Kimono",
+    "Windsor_tie": "eine Windsor-Krawatte",
+    "ladle": "Schöpfkelle",
+    "cornet": "ein Kornett",
+    "punching_bag": "ein Boxsack",
+    "tripod": "ein Dreifuß",
     "cellular_telephone": "ein Handy",
     "modem": "ein Modem",
     "computer_keyboard": "eine Tastatur",
     "space_bar": "eine Leertaste",
     "consomme": "eine Brühe",
     "cup": "eine Tasse",
     "espresso": "ein Espresso",
-    "face_powder": "Gesichtspuder",
-    "soap_dispenser": "Seifenspender",
+    "face_powder": "ein Gesichtspuder",
```

Comment:
```suggestion
    "face_powder": "Gesichtspuder",
```
Makes more sense this way, I think.